npm/mount-utils/README.md

```diff
@@ -2,7 +2,7 @@
 
 > **Note** this package is not meant to be used outside of cypress component testing.
 
-This librares exports some shared types and utility functions designed to build adapters for components frameworks.
+This library exports some shared types and utility functions designed to build adapters for components frameworks.
```

Comment:
Might be worth documenting `getContainerEl()` since it's now part of the public API (we had to expose it and make it official, since you cannot make your own `unmount` without it).

https://github.com/cypress-io/cypress/blob/develop/npm/mount-utils/src/index.ts#L3-L13

I think it would make sense to be a dot point under the `Requirements` heading in this README.